db/migrate/20200329222246_add_high_priority_column_to_notifications.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class AddHighPriorityColumnToNotifications < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+  def up
+    if !column_exists?(:notifications, :high_priority)
+      add_column :notifications, :high_priority, :boolean, default: nil
+    end
+
+    # type 6 = private message, 24 = bookmark reminder
+    # priority 0 = low, 1 = normal, 2 = high
+    execute <<~SQL
+      UPDATE notifications SET high_priority = TRUE WHERE notification_type IN (6, 24);
```

Comment:
hmmm I would still keep it to one statement per execute, that way if it blows up it will be easier to find out where it exploded